src/main/java/exception/CommandException.java

```diff
@@ -0,0 +1,12 @@
+package exception;
+
+/**
+ * Represents an exception which is thrown if the command is not known.
+ */
+public class CommandException extends DukeException {
```

Comment:
I like how you have separated the DukeException into smaller exceptions. Made reading easier!